test/devise/passkeys/controllers/concerns/test_reauthentication.rb

```diff
@@ -69,22 +69,22 @@ def resource_name
     assert_nil @test_class.session["test_value:1234_current_reauthentication_token"]
   end
 
-  test "#passkey_reauthentication_token_key" do
-    assert_equal "test_value:1234_current_reauthentication_token", @test_class.passkey_reauthentication_token_key
+  test "#Reauthentication_token_key" do
+    assert_equal "test_value:1234_current_reauthentication_token", @test_class.Reauthentication_token_key
```

Comment:
These need to be reverted back to lowercase, because Ruby methods are generally lowercase